zephyr/dts/riscv/tinyvision/boot_0x200000.dtsi

```diff
@@ -12,6 +12,20 @@
 		zephyr,flash = &flash0; 
 		zephyr,flash-controller = &flashctrl;
 	};
+
+	soc {
+		compatible = "tinyvision,tinyclunx33";
+		ranges;
+		#address-cells = <1>;
+		#size-cells = <1>;
```

Comment:
That should not cause a problem as this is set the same way as in `tinycunx33_rtl.dtsi`, but you can remove these lines as this is already set elsewhere.

Devicetree acts as a way to insert/replace/delete data in an unique tree per system.